src/cmake/compiler.cmake

```diff
@@ -283,6 +283,33 @@ if (NOT USE_SIMD STREQUAL "")
     add_compile_options (${SIMD_COMPILE_FLAGS})
 endif ()
 
+###########################################################################
+# Batched SIMD shader execution options.
+#
+# The USE_BATCHED option may be set to indicate that support for batched
+# SIMD shader execution be compiled along with targe specific libraries
+set (USE_BATCHED "" CACHE STRING "Build batched SIMD shader execution for (0, b8_AVX, b8_AVX2, b8_AVX2_noFMA, b8_AVX512, b8_AVX512_noFMA, b16_AVX512, b16_AVX512_noFMA)")
+set (BATCHED_SUPPORT_DEFINES "")
+set (BATCHED_TARGET_LIBS "")
+set (BUILD_BATCHED False)
+if (NOT USE_BATCHED STREQUAL "")
+    message (STATUS "Compiling with batched SIMD targets ${USE_BATCHED}")
+    if (NOT USE_BATCHED STREQUAL "0")
+        add_compile_definitions ("OSL_USE_BATCHED=1")
```

Comment:
I think you won't need this add_compile_definitions if you do the other thing I suggested with oslconfig.h.